### PR TITLE
[ISSUE #16] [ISSUE #27] [ISSUE #32] Open Resource Limit and HostNetwork Configurations in CRD 

### PR DESCRIPTION
**Pull request by liuruiyiyang**
Change List:
1.Fix issue #27 
2.Fix issue #32 
3.Fix issue #16 
4.Change uid and gid from 3000 to root in name server and broker images.
5.Optimize broker waiting for all name server instances ready mechanism.
6.Put name-server and broker yaml together in example
7.Revise README docs accordingly.

### CODE REVIEW

File: example/rocketmq_v1alpha1_broker_cr.yaml [hunk 1]
```diff
@@ -31,10 +31,18 @@ spec:
   brokerImage: rocketmqinc/rocketmq-broker:4.5.0-alpine
```
Comment (vongosling) [inline example/rocketmq_v1alpha1_broker_cr.yaml:31]: ?

File: example/rocketmq_v1alpha1_nameservice_cr.yaml [hunk 2]
```diff
@@ -24,8 +24,25 @@ spec:
   nameServiceImage: rocketmqinc/rocketmq-namesrv:4.5.0-alpine
```
Comment (vongosling) [inline example/rocketmq_v1alpha1_nameservice_cr.yaml:24]: ?

File: README.md [hunk 3]
```diff
@@ -204,21 +166,29 @@ metadata:
...
   # nameServers is the [ip:port] list of name service
   nameServers: ""
   # replicationMode is the broker replica sync mode, can be ASYNC or SYNC
   replicationMode: ASYNC
-  # replicaPerGroup is the number of replica broker in each group
+  # replicaPerGroup is the number of each broker cluster
   replicaPerGroup: 1
   # brokerImage is the customized docker image repo of the RocketMQ broker
-  brokerImage: apacherocketmq/rocketmq-broker:4.5.0-alpine
+  brokerImage: rocketmqinc/rocketmq-broker:4.5.0-alpine
```
[the remark this answers is not in the document]
Comment (liuruiyiyang) [reply README.md]: fixed, changed to apacherocketmq

File: example/rocketmq_v1alpha1_broker_cr.yaml (hunk 1 above)
Comment (liuruiyiyang) [reply example/rocketmq_v1alpha1_broker_cr.yaml]: fixed

File: example/rocketmq_v1alpha1_nameservice_cr.yaml (hunk 2 above)
Comment (liuruiyiyang) [reply example/rocketmq_v1alpha1_nameservice_cr.yaml]: fixed

**Pull request merged**